doc/usage.md

```diff
@@ -576,11 +576,15 @@ error result.
 Compliance Operator is able to run a platform scan on the HyperShift Managment cluster
 for the Hosted Cluster with a tailoredProfile.
```

Comment:
A picture describing, or better, linking to the architecture of hypershift would be useful.

Reading this from the perspective of someone who doesn't know what hypershift is, it's hard to discern what "hosted" versus "management" clusters are.

We can link to https://hypershift-docs.netlify.app/reference/concepts-and-personas/ to describe those concepts and reuse them here.

Reply:
Agree, what do you think about https://github.com/ComplianceAsCode/compliance-operator/pull/382/commits/90fbb120ea0d05960485e4ac9c3c1574a20ad823 ?